libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -212,7 +213,17 @@ void elasticsearch_plugin_impl::doOperationHistory(const optional <operation_his
    os.op_in_trx = oho->op_in_trx;
    os.operation_result = fc::json::to_string(oho->result);
    os.virtual_op = oho->virtual_op;
-   os.op = fc::json::to_string(oho->op);
+
+   if(_elasticsearch_operation_object) {
+      oho->op.visit(fc::from_static_variant(os.op_object, FC_PACK_MAX_DEPTH));
+      if (os.op_object.is_object()) {
```

Comment:
I don't see operations that are not objects, added an `else` with a log to check if it was happening at some point. Synchronized a decent part of the chain without hitting the log.
If it happen for any reason i think it will be ok to avoid visiting the adaptor completely, rest of the data will be inserted anyways. 
Unsure if remove the check or not.

Reply:
IMO remove it. We already know that ``oho->op`` is a valid operation object at this point. It will be serialized using ``fc::reflect``, which always creates an ``fc::variant_object``.